shell/platform/darwin/macos/framework/Source/FlutterViewController_Internal.h

```diff
@@ -5,18 +5,24 @@
 #import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterViewController.h"
 
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterKeySecondaryResponder.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.h"
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterView.h"
 
 @interface FlutterViewController ()
 
 // The FlutterView for this view controller.
-@property(nonatomic, readonly, nullable) FlutterView* flutterView;
+@property(nonatomic, strong, nullable) FlutterView* flutterView;
```

Comment:
I know this is the internal header, but I think we probably still want this ot be `readonly` in the header. We can redefine it as readwrite in the implementation file instead.